defactor-com/ui-kit
# Fix/Ui Kit Issues

## Ui Kit Issues

### What does this PR do?

- Resolve #207 
- Resolve #208 
- Resolve #209 
- Resolve #210
- Resolve #212

### Steps to test

1. Run project
2. Check changes

Now:
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/6f2707b8-96a0-4368-8935-ed2495dcd00a)

#### CheckList
